src/components/settings/Settings.h

```diff
@@ -308,6 +320,8 @@ namespace Pinetime {
         uint16_t shakeWakeThreshold = 150;
 
         Controllers::BrightnessController::Levels brightLevel = Controllers::BrightnessController::Levels::Medium;
+
+        DfuAndFsMode dfuAndFsMode = DfuAndFsMode::Disabled;
```

Comment:
If the mode EnabledTillReboot is saved into the settings, and restored on reboot then it is effectively always enabled. Please check my suspicion

Reply:
See https://github.com/InfiniTimeOrg/InfiniTime/pull/1891/files#r1807283475

Reply:
@NeroBurner The `DfuAndFsMode::EnabledTillReboot` should never be restored on reboot. Please check the link from @mark9064's comment above.